docs/program-setup/bare_metal/c_small_libs.md

```diff
@@ -1,5 +1,7 @@
 <h1 id="using-small-c-libraries">Using small C libraries in Mbed OS bare metal</h1>
-
+<!--is this page for app developers? do we expect them to always manually specify for their app that they want to use bare metal?-->
+<!--I'm also intriggued by the phrasing "if your application doesn't use an RTOS". Of course it doesn't; we're talking about bare metal-->
```

Comment:
OK- sentence needs rephrasing. The point was to say "1- use bare metal profile if you don't need a RTOS 2- use small library with bare metal profile"

Reply:
So if I have to use this in bare metal apps, should it be part of - or at least mentioned - in the new tutorial in PR https://github.com/ARMmbed/mbed-os-5-docs/pull/1305?